cli/server/compare

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+ARG1=$1
+ARG2=$2
+if [[ -z $ARG1 || ! -d $ARG1 ]]; then
+	echo first directory is not provided
+	exit 1
+fi
+
+if [[ -z $ARG2 || ! -d $ARG2 ]]; then
+	echo second directory is not provided
+	exit 1
+fi
+
+FIRST=$3
+if [[ -z "$FIRST" ]]; then
+    FIRST=1
+fi
+
+LAST=$4
+if [[ -z "$LAST" ]]; then
+    LAST=40 # 40_00000 block
+fi
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+# directories contain 100k blocks
+for i in `seq $FIRST $LAST`; do
+	dir=BlockStorage_${i}00000
+	echo Processing directory $dir
+
+	# files are grouped by 1k blocks
+	for j in `seq $(((i-1)*100 + 1)) $((i*100))`; do
+		file=dump-block-${j}000.json
+		"$SCRIPT_DIR"/compare-json-dumps $ARG1/$dir/$file $ARG2/$dir/$file
```

Comment:
I don't think it's worth making it a separate script. If you ever need to compare two files directly just make this script work in this fashion also (it can be done automagically, based on parameters). Also, `cli/server` is a good directory for go code, but I don't expect to see any bash scripts there. Top-level `scripts`, `util` or `bin` would be more appropriate.